PubNubUnity/Assets/EndPoints/PubSub/SubscribeBuilder.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace PubNubAPI
+{
+    public class SubscribeBuilder
+    {
+        private readonly SubscribeRequestBuilder pubBuilder;
+        public SubscribeBuilder(PubNubUnity pn){
+            pubBuilder = new SubscribeRequestBuilder(pn);
+        }
+        
+
+        #region IPubNubBuilder implementation
+
+        public void Execute(){
+            pubBuilder.Execute();
+        }
+
+        public SubscribeBuilder WithPresence(){
+            pubBuilder.WithPresence();
+            return this;
+        }
+
+        public SubscribeBuilder SetChannels(List<string> channels){
```

Comment:
Most of the other builders seem to use Channels (and other setters) without the Set-prefix.

Reply:
thats true, modifying. Thanks @schakkis !